test/lib/RuleManager.test.ts

```diff
@@ -240,7 +240,7 @@ describe('RuleManager', () => {
 						results = JSON.parse(output);
 					}
 
-					expect(results).to.be.an("array").that.has.length(4);
+					expect(results).to.be.an("array").that.has.length(6);
```

Comment:
I verified the math on this, it's a combination of removing some PMD js results and adding in the `Error Prone` results